spec/forms/response_form_spec.rb

```diff
@@ -5,25 +5,50 @@
 RSpec.describe ResponseForm do
   describe '#save' do
     context 'given valid attributes' do
-      it 'returns true' do
-        questions_ids = [
-          { id: '940d229e4cd87cd1e202' },
-          { id: 'c3a9b8ce5c2356010703' }
-        ]
-        form = described_class.new(survey_id: 'd5de6a8f8f5f1cfe51bc', questions: questions_ids)
-
-        expect(form.save).to eq(true)
+      context 'given a same questions order' do
```

Comment:
Should it be something like this?
```suggestion
      context 'given a questions_ids param that has the same order as questions in DB' do
```

Reply:
And same for the below 🙏

Reply:
Fixed in https://github.com/nimblehq/nimble-survey-web/pull/82/commits/c9c6b71e2065341d7ce38729584639dfec5ed152